src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  color: var(--dark-color);
+}
+
+.card__container {
+  max-width: 200px;
+  padding: 32px 16px 0;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  display: block;
+  margin: 0 auto 40px;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-title {
+  margin-bottom: 4px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  line-height: 14px;
+
+  font-size: 10px;
+  color: var(--grey-color);
+}
+
+.card__product-rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__product-reviews {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__product-price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  color: var(--grey-color);
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button {
+  display: block;
+  width: 166px;
```

Comment:
Try this for .card__button:   width: 100%;  padding: 10.5px 0;
Then run tests

Reply:
agree